src/moneypenny/handlers/external/agent.py

```diff
@@ -36,23 +37,26 @@ async def check_status(request: web.Request) -> web.Response:
      a 202 response if it is still in progress, and raises an error otherwise.
     If the pod cannot be found, a 404 is raised; otherwise a 500 with
      descriptive text is raised.
+
+    Because Moneypenny tidies up rather quickly, a 200 is very unlikely,
+     a 202 fairly unlikely, and a 404 common.
```

Comment:
Given this, is there much point in this route? I'm not sure when I would use it or why.

Reply:
Because we submit the request and don't wait for it to complete.  So if you wanted to write a UI around Moneypenny, for instance, you'd submit a request and then show a spinner while you got a 202, returning "it completed" when you get a 200 or a 404.